index.html

```diff
@@ -119,6 +119,7 @@
         }
         .scrollable-contrib-list:hover, .scrollable-contrib-list:active, .scrollable-contrib-list:focus {
           overflow: auto;
+		  overflow-y: overlay;
```

Comment:
Nit: we indent using spaces, not tabs
```suggestion
          overflow-y: overlay;
```